core/src/main/java/com/graphhopper/reader/dem/AbstractElevationProvider.java

```diff
@@ -68,35 +72,44 @@ void setSleep(long sleep) {
         this.sleep = sleep;
     }
 
-    @Override
-    public void setAutoRemoveTemporaryFiles(boolean autoRemoveTemporary) {
-        this.autoRemoveTemporary = autoRemoveTemporary;
-    }
-
-    public void setDownloader(Downloader downloader) {
-        this.downloader = downloader;
-    }
-
-    protected File getCacheDir() {
-        return cacheDir;
-    }
-
-    @Override
-    public ElevationProvider setBaseURL(String baseUrl) {
+    /**
+     * For the MultiSourceElevationProvider you have to specify the base URL separated by a ';'.
+     * The first for cgiar, the second for gmted.
+     */
+    public AbstractElevationProvider setBaseURL(String baseUrl) {
```

Comment:
Can we keep the less specific javadoc of this method that we had in `ElevationProvider` here and put this more specific javadoc at the override in `MultiSourceElevationProvider`?